terraform/applications/careers/static.tf

```diff
@@ -0,0 +1,189 @@
+# all the relevant infra for the static site
+resource "aws_s3_bucket" "logs" {
+  bucket = "mozilla-careers-logs-783633885093"
```

Comment:
Should the bucket name say "stage"? (If we're sharing the bucket with prod but at another pathname, see also the force_destroy question following.)

Reply:
I've switched to a stage log bucket.  I think you pay per what you store, so there shouldn't be any cost reason to prefer 1 shared bucket over two.  In the previous version of careers, both stage and prod were defined in the same file, so sharing the logging bucket was easy.